frontend/src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -450,7 +450,15 @@ export default defineComponent({
        * runtime checks.
        */
       if (localAudio) {
-        localAudio.currentTime = frac * duration.value
+        if (Number.isFinite(duration.value)){
+          localAudio.currentTime = frac * duration.value
+        }
+        else if (Number.isFinite(localAudio.duration)){
+          localAudio.currentTime = frac * localAudio.duration
+        }
+        else {
+          console.warn('Duration is not finite. Skipping seek.')
```

Comment:
@obulat should this use `warn` imported from `~/utils/logger`?

Reply:
Hi @sarayourfriend,

I noticed that the use `warn` isn't imported from `~/utils/logger`. Should I import it, or do you want me to remove the `warn` usage?

Thanks!

Reply:
Oops, I mean `~/utils/console`, not `logger`, apologies.

But. it would need to be imported, yes. I just wanted to check with Olga whether it should be used here rather than `console.warn` directly. I don't know whether we specifically want to always log this on the client for some reason, for example, which `~/utils/console` would suppress.

Reply:
@obulat can you review this discussion and make an issue if necessary? Thanks!